setup.cfg

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2021 Graz University of Technology.
+#
+# invenio-campusonline is free software; you can redistribute it and/or
+# modify it under the terms of the MIT License; see LICENSE file for more
+# details.
+
+
+[build_sphinx]
+source-dir = docs/
+build-dir = docs/_build
+all_files = 1
+
+[bdist_wheel]
+universal = 1
+
+[pydocstyle]
+add_ignore = D401
+
+[pycodestyle]
+exclude = docs/conf.py
+
+[compile_catalog]
+directory = invenio_campusonline/translations/
+
+[extract_messages]
+copyright_holder = Graz University of Technology
+msgid_bugs_address = info@tugraz.at
+mapping-file = babel.ini
+output-file = invenio_campusonline/translations/messages.pot
+add-comments = NOTE
+
+[init_catalog]
+input-file = invenio_campusonline/translations/messages.pot
+output-dir = invenio_campusonline/translations/
+
+[update_catalog]
+input-file = invenio_campusonline/translations/messages.pot
+output-dir = invenio_campusonline/translations/
```

Comment:
could you please add the following to this file. so we have on all our repos the same configuration
[flake8]
max-line-length = 88
extend-ignore = E203
select = C,E,F,W,B,B950
ignore = E501

[isort]
multi_line_output = 3
include_trailing_comma = True
force_grid_wrap = 0
use_parentheses = True
ensure_newline_before_comments = True
line_length = 88

[pycodestyle]
ignore = E203,E501